src/load-nyc-config-sync.js

```diff
@@ -2,13 +2,16 @@
 'use strict'
 
 const { loadNycConfig } = require('@istanbuljs/load-nyc-config')
+const isElectron = process.versions && process.versions.electron
 
 async function main () {
   const [cwd, nycrcPath] = process.argv.slice(2)
 
   console.log(JSON.stringify(await loadNycConfig({ cwd, nycrcPath })))
+  if (isElectron) process.exit(0)
 }
 
 main().catch(error => {
   console.log(JSON.stringify({ 'load-nyc-config-sync-error': error.message }))
+  if (isElectron) process.exit(1)
```

Comment:
```suggestion
  process.exit(0)
```

This may seem odd but the babel plugin looks at stdout for the error message, I'm not sure but exiting with code 1 might cause problems and cause the babel plugin to see a spawn error rather than seeing the config load error.